### PR TITLE
Sscs 9061 allow social fund

### PR DESCRIPTION
**Pull request by jhart-dev**
added addresses for Social Fund benefit type

Jira Ticket: [https://tools.hmcts.net/jira/browse/SSCS-9061](url)

### CODE REVIEW

**Review by jackmaloney — APPROVED** [review]

File: src/main/resources/reference-data/dwpAddresses.json [hunk 1]
```diff
@@ -571,5 +571,51 @@
...
+        "line2": "Wolverhampton",
+        "postCode": "WV98 1SU"
+      }
+    },
+    {
+      "code": "Funeral Payment Dispute Resolution Team",
+      "isDefault": false,
+      "mapping": {
+        "ccd": "Funeral Payment Dispute Resolution Team",
+        "gaps": "Funeral Payment",
```
Comment (danielbe11) [inline src/main/resources/reference-data/dwpAddresses.json:595]: Should be "Funeral Payments DRT". See SSCS-9067 ticket comments.

File: src/main/resources/reference-data/dwpAddresses.json [hunk 2]
```diff
@@ -571,5 +571,51 @@
...
+        "dwpRegionCentre": "JSA Birkenhead"
```
Comment (danielbe11) [inline src/main/resources/reference-data/dwpAddresses.json:596]: Should be "Funeral Payment". See SSCS-9067 ticket comments.

File: src/main/resources/reference-data/dwpAddresses.json [hunk 3]
```diff
@@ -571,5 +571,51 @@
...
+      "code": "Pensions Dispute Resolution Team",
+      "isDefault": false,
+      "mapping": {
+        "ccd": "Pensions Dispute Resolution Team",
+        "gaps": "Pensions Disputes Resolution Team",
+        "dwpRegionCentre": "Winter Fuel Payment"
+      },
+      "address": {
+        "line1": "The Pension Service 11",
+        "line2": "Mail handling site A",
```
Comment (danielbe11) [inline src/main/resources/reference-data/dwpAddresses.json:615]: Should be "Mail Handling Site A".

File: src/main/resources/reference-data/dwpAddresses.json [hunk 4]
```diff
@@ -571,5 +571,51 @@
...
+      "mapping": {
+        "ccd": "Funeral Payment Dispute Resolution Team",
+        "gaps": "Funeral Payment",
+        "dwpRegionCentre": "JSA Birkenhead"
+      },
+      "address": {
+        "line1": "The Pension Service 11",
+        "line2": "Mail Handing Site A",
+        "line3": "Wolverhampton",
+        "postCode": "WV99 1FZ"
```
Comment (danielbe11) [inline src/main/resources/reference-data/dwpAddresses.json:602]: Should be "WV98 1LW". See SSCS-9067 ticket Description for the address.

**Pull request merged**